x-pack/plugins/observability_solution/investigate_app/server/index.ts

```diff
@@ -4,21 +4,32 @@
  * 2.0; you may not use this file except in compliance with the Elastic License
  * 2.0.
  */
-import type { PluginInitializer, PluginInitializerContext } from '@kbn/core/server';
-import { InvestigateAppConfig } from './config';
-
+import type {
+  PluginConfigDescriptor,
+  PluginInitializer,
+  PluginInitializerContext,
+} from '@kbn/core/server';
+import type { InvestigateAppConfig } from './config';
 import { InvestigateAppPlugin } from './plugin';
 import type {
   InvestigateAppServerSetup,
   InvestigateAppServerStart,
   InvestigateAppSetupDependencies,
   InvestigateAppStartDependencies,
 } from './types';
+import { config as configSchema } from './config';
 
 export type { InvestigateAppServerRouteRepository } from './routes/get_global_investigate_app_server_route_repository';
 
 export type { InvestigateAppServerSetup, InvestigateAppServerStart };
 
+export const config: PluginConfigDescriptor<InvestigateAppConfig> = {
+  schema: configSchema,
+  exposeToBrowser: {
+    enabled: true,
```

Comment:
Should this be true by default?

Reply:
You mean I could remove this and the config would be exposed to the public plugin?

Reply:
No, I thought maybe this is also similar to a feature flag that we need to enable ourselves. Maybe I am mistaken, what does this do? Is it OK to have it enabled by default?

Reply:
Ah no, this is just to expose the "xpack.investigateApp.enabled" config to the public plugin, otherwise it's not accessible there. The `true` is just for exposing the config, regardless of the config's value